tests/media/client/ipc/ipcClient/IpcClientTest.cpp

```diff
@@ -132,5 +134,12 @@ TEST_F(IpcClientTest, UnexpectedDisconnectWithNotification)
     {
     }
 
+    // Wait for the callback
+    constexpr unsigned int kWaitTimeMilliseconds = 5000;
+    for (unsigned int i = 0; !connectionBrokenCallbackCalled && i < kWaitTimeMilliseconds; ++i)
```

Comment:
I would expect us to use the wait_for c++ method here along with a condition variable triggered when onConnectionBroken gets called alongstide this bool.

Also, can this test be refactored so that we only have one wait/cond variable? so basically move the m_eventsCond and m_eventsLock to be used here instead.